### ISSUE TITLE
Extract the feature

### ISSUE THREAD

**Issue by nemo0526** [hbchen121/AICITY2022_Track2_SSM#6]
How can I get the extracted feature when every training?I can't actually understand the instructions in Ensemble and Submit step.It seems do the same thing in the training step.

thanks again for your replying!

**Comment by hbchen121** [comment]
The test set is Valid set when you training the model, and the test set is Test set in online judgement system when you sumbit the results. When you reproduce the project, the online system has been closed. So you don't need to pay attention to the submit, and the model ensemble is used for Test set. You only need to focus on the performance of the single model in training stage.
